src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.CURLcode.cs

```diff
@@ -12,6 +12,7 @@ internal enum CURLcode
             CURLE_UNSUPPORTED_PROTOCOL  =  1,
             CURLE_NOT_BUILT_IN = 4,
             CURLE_COULDNT_RESOLVE_HOST  =  6,
+            CURLE_ABORTED_BY_CALLBACK = 42,
```

Comment:
I grepped ```CURLE_ABORTED``` in the native folder, found nothing and concluded that this was not necessary. :smile:  It seems a couple of others were also missing in the PAL enums. I'll add them too.